framework/auth/views.py

```diff
@@ -150,7 +152,7 @@ def auth_login(auth, **kwargs):
 
     status_message = request.args.get('status', '')
     if status_message == 'expired':
-        status.push_status_message('The private link you used is expired.')
+        status.push_status_message('The private link you used is expired.  Please Resend email.')
```

Comment:
Lowercase "resend". Also, could the message guide the user a little more? Like "Please click the *Resend email* button" or something similar?